SearchIncidentsV2 - fixed encoding error (#11990)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_3_31.md

#### Scripts
##### SearchIncidentsV2

> - Fixed an issue where the value of an incident id from type int raised a TypeError exception.

Reviewer: ```suggestion
- Fixed an issue where the value of an incident ID was an integer and would raise a TypeError exception.
```

> - Updated the Docker image to demisto/python3:3.9.2.17957.

Reviewer: ```suggestion
- Updated the Docker image to: *demisto/python3:3.9.2.17957*.
```
